src/__tests__/StateManaged.test.js

```diff
@@ -0,0 +1,612 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import cases from 'jest-in-case';
+
+import { OPTIONS } from './constants';
+import Select from '../';
+import { components } from '../components';
+
+const { Control, Menu, MultiValue, Placeholder, Option } = components;
+
+const BASIC_PROPS = { options: OPTIONS, name: 'test-input-name' };
+
```

Comment:
Please add test cases for validating the expected behaviour of exposed controlled props: 
1. props.inputValue
2. props.menuIsOpen 
3. props.value

Reply:
done